app/services/analytics_service/celery_worker.py

```
@@ -0,0 +1,202 @@
+            concurrency=concurrency,
+            queues=queues,
+            log_level=log_level,
+            redis_host=self.settings.redis_host,
+            redis_port=self.settings.redis_port,
+            redis_db=self.settings.redis_db
+        )
+        
+        # Configure Celery app
+        celery_app.conf.update(
+            worker_log_format='[%(asctime)s: %(levelname)s/%(processName)s] %(message)s',
+            worker_task_log_format='[%(asctime)s: %(levelname)s/%(processName)s][%(task_name)s(%(task_id)s)] %(message)s',
```

Pylint (3.3.7)

[convention] Line too long (122/100)

(C0301)